### PR TITLE
Document how to delete all sessions for both file and redis storage

### PR DESCRIPTION
**Pull request by inkhey**
related to #3988 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link) : not relevant
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [X] Manual, quality tests have been done : not relevant

### CODE REVIEW

File: backend/doc/setting.md [hunk 1]
```diff
@@ -188,6 +191,14 @@ To avoid keeping expired session files you should run :
 regularly (for example by using a cron job), which will delete file which have not been modified since 10 days.
 You should use this command in both session data and session lock dirs.
 
+## delete all existing sessions
+
+```shell
+# note: <session.data_dir> refer to absolute path given by config parameter session.data_dir
```
Comment (grignards) [inline backend/doc/setting.md:197]:
```suggestion
# note: <session.data_dir> refers to the absolute path given by the config parameter `session.data_dir`.
```

File: backend/doc/setting.md [hunk 2]
```diff
@@ -198,6 +209,14 @@ Then you'll need to set those parameters for redis backend:
     session.type = ext:redis
     session.url = redis://localhost:6379/0
 
+
+## delete all existing sessions
```
Comment (grignards) [inline backend/doc/setting.md:213]:
```suggestion
## delete the existing sessions
```

File: backend/doc/setting.md [hunk 3]
```diff
@@ -198,6 +209,14 @@ Then you'll need to set those parameters for redis backend:
...
+
+```shell
+# note: <session.url> refer to value of config parameter session.url (redis url)
```
Comment (grignards) [inline backend/doc/setting.md:216]:
```suggestion
# note: <session.url> refers to the value of the config parameter `session.url` (redis url)
```

**Review by PhilippeAccorsi — COMMENTED** [review]
Same title visible two time (on line 194 and 213). + tree line visible before line 202
I not really understand organisation of all part.

File: backend/doc/setting.md [hunk 4]
```diff
@@ -188,6 +190,14 @@ To avoid keeping expired session files you should run :
 regularly (for example by using a cron job), which will delete file which have not been modified since 10 days.
 You should use this command in both session data and session lock dirs.
 
+## delete all existing sessions
```
Comment (inkhey) [inline backend/doc/setting.md:193]:
```suggestion
#### delete all existing sessions (file storage)
```

File: backend/doc/setting.md [hunk 5]
```diff
@@ -198,6 +208,14 @@ Then you'll need to set those parameters for redis backend:
     session.type = ext:redis
     session.url = redis://localhost:6379/0
 
+
+## delete the existing sessions
```
Comment (inkhey) [inline backend/doc/setting.md:212]:
```suggestion
#### delete the existing sessions (redis storage)
```

**Comment by inkhey** [comment]
> Same title visible two time (on line 194 and 213). + tree line visible before line 202
> I not really understand organisation of all part.

I did not properly set tree level. is it better this way ?
I also add precise title in order to help user find if they are in the good section.

**Review by PhilippeAccorsi — APPROVED** [review]

**Review by grignards — APPROVED** [review]

**Pull request merged**